src/util/postgres_client.js

```diff
@@ -802,7 +806,17 @@ class PostgresTable {
         let mr_q;
         let query_string;
         try {
-            query_string = `SELECT * FROM ${this.name} WHERE ${mongo_to_pg('data', encode_json(this.schema, options.query), {disableContainmentQuery: true})}`;
+            let where_clause = mongo_to_pg('data', encode_json(this.schema, options.query));
```

Comment:
I suggest moving it to a function and calling it from other relevant flows. e.g (db_clener).